proto/relayer/chains/ethereum/config/config.proto

```diff
@@ -23,4 +23,7 @@ message ChainConfig {
 
   uint64 average_block_time_msec = 9;
   uint64 max_retry_for_inclusion = 10;
+
+  double gas_estimate_multiplier = 11;
```

Comment:
Sorry for lack of my explanation.
In such a case, please define a fraction type and use it instead of double.
ref: https://github.com/hyperledger-labs/yui-relayer/blob/c226f1b1cee1c8da62e04b7929ed24b4b74bc125/proto/relayer/chains/tendermint/config/config.proto#L25-L28

Reply:
Change to Fraction